Add donate page - DO NOT MERGE YET!

Vaults1 opened the pull request:
Please review it.
I have no idea how you edit the background, so if anyone knows how feel free to edit my PR.

**Note to BOD staff:** This PR allows edits from maintainers, please make use of it. Thanks!

Commit:
Create donate.html

Commit:
Delete CNAME

Commit:
Update donate.html

Commit:
Update donate.html

Commit:
Update donate.html

Commit:
Create CNAME

InspectorGadget reviewed (commented):
Everything looks great btw, i'd suggest using .php extension at the back so it'd look even fancier. I know for the fact that changing it isn't hard. Another suggestion from me is that made a separate dir for each and every page, this could stop being inspect elemented or 'view page source'.

AvgZing reviewed (approved):
Awesome job! I think this should be merged immediately, and the reasons for why to donate are great! I couldn't have said it better myself!

NiekertDev reviewed (approved):
Good work! Merge it as fast as possible!

Ad5001 reviewed (approved):
Great job I like it !

AvgZing commented:
@RTGDaCoder We can't make it .php because github pages does not support php. We also don't care at all about if they can inspect element because all of the code is open source anyways.

AvgZing commented:
Just waiting for these four team members!
@Dog2puppy
@Taki21
@HimbeersaftLP 
@TheDragonRing

lavender-sys commented:
I can never find the review button @github please make it easier to find the review button.

HimbeersaftLP commented:
> Everything looks great btw, i'd suggest using .php extension at the back so it'd look even fancier. I know for the fact that changing it isn't hard. Another suggestion from me is that made a separate dir for each and every page, this could stop being inspect elemented or 'view page source'.

I don't want to offend you, but did you even notice what crap you just said? lol

HimbeersaftLP committed:
You don't need to import jquery two times lol

lavender-sys commented:
@HimbeersaftLP I don't understand what he means by people can inspect element.

HimbeersaftLP commented:
@Dog2puppy Yeah, ppl can always do inspect element, there is no way to hide the source of a html page (flash player doesn't count as html lol)

lavender-sys commented:
Who uses Flash Player in 2017?

HimbeersaftLP commented:
@Dog2puppy Some crappy website building tools

lavender-sys commented:
XD

HimbeersaftLP reviewed (commented):
I think that loading gif should be replaced with a transparent one (but please take care of copyright when downloading one 😛 )

Vaults1 commented:
@HimbeersaftLP I'm trying to find out how to change the background 😜

HimbeersaftLP commented:
@Ultie0 Which background?

Vaults1 commented:
@HimbeersaftLP the one behind the text, idk how to make it white.

HimbeersaftLP commented:
It depends on in which div you put it

Thunder33345 commented:
have to say this, GITHUB IS A STATIC PAGE HOST NOT WEB HOST
but you can use javascript doeeeee

lavender-sys committed:
Donate page link. 

If we used jekyll, it would be so much easier when we add pages because we would have a footer and a header template.

HimbeersaftLP commented on index.html:
```diff
@@ -105,7 +106,7 @@ <h4 style="margin-left: 20px;">Staff Members</h4>
             <i class="fa fa-cog"></i>
             <h4>Highly Customizeable</h4>
-            <p>All of our products are able to be customized through their various settings files. We try to maximize customizeability (is that a word? 😋).</p>
+            <p>All of our products are able to be customized through their various settings files. We try to maximize customizeability (is that a word? ).</p>
```
Ya broke all da poor emojis 😢

lavender-sys replied:
I guess chrome doesnt like emojis in text areas. I'll fix it.

lavender-sys replied:
And I dont know how to fix it... @Ultie0 @TheRoyalBlock @someone please help me

AvgZing replied:
I think i did it

Commit:
lol

Commit:
XD

AvgZing reviewed (dismissed):
Please just merge approve this commit. EMOJIS!

dragonwocky reviewed (commented):
Go ahead and merge. Looks good!

AvgZing commented:
We have a 5 out of 8 vote! Majority rules! I'll let @Ultie0 do the honors.

lavender-sys commented:
Yay!

AvgZing commented:
WOOT!
